state/model.go

```diff
@@ -685,7 +687,11 @@ func (m *Model) StatusHistory(filter status.StatusHistoryFilter) ([]status.Statu
 
 // Config returns the config for the model.
 func (m *Model) Config() (*config.Config, error) {
```

Comment:
How does a generation on a model config work? I wouldn't have expected it based on the spec.

Reply:
I will seek clarification there. I took it as implied from:

> Which model operations are tracked via generations?
> Not all changes to the model are tracked in this way. 
> - upgrade-charm
> - config
> - attach